### ISSUE TITLE
add example for CompoundLocation in tutorial

### ISSUE THREAD

**Issue by timflutre** [biopython/biopython#648]
I managed to find how to iterate over a CompoundLocation, but it wasn't that easy. So maybe one could add the following example to the tutorial (file chapter_seq_annot.tex, section "CompoundLocation object")? ([source](http://biopython.org/DIST/docs/api/Bio.SeqFeature.CompoundLocation-class.html#_flip))

>>> from Bio.Seq import Seq
>>> from Bio.SeqFeature import FeatureLocation
>>> dna = Seq("nnnnnAGCATCCTGCTGTACnnnnnnnnGAGAMTGCCATGCCCCTGGAGTGAnnnnn")
>>> small = FeatureLocation(5, 20, strand=1)
>>> large = FeatureLocation(28, 52, strand=1)
>>> location = small + large
>>> print(small)
[5:20](+)
>>> print(large)
[28:52](+)
>>> print(location)
join{[5:20](+), [28:52](+)}
>>> for part in location.parts:
...     print(len(part))
...
15
24

**Comment by peterjc** [comment]
Good point. This is all covered in the built-in docstring help (e.g. ``help(FeatureLocation)``) also online at http://biopython.org/DIST/docs/api/Bio.SeqFeature.CompoundLocation-class.html but the Tutorial makes no reference to that currently.

Would that suffice?

**Comment by timflutre** [comment]
Ok, I see. I don't use Python interactively very often, but in such cases I should.

I guess it's better than nothing? ;)
